plugin.yaml

```diff
@@ -1096,3 +1098,156 @@ workflows:
         description: >
           List of calls to run
         default: []
+
+  wrap_start:
```

Comment:
Change name to `alt_start` ... and all the others to `alt_...` What do you think?